project/ProjectAutoPlugin.scala

```diff
@@ -22,7 +22,7 @@ object ProjectAutoPlugin extends AutoPlugin {
       developers += Developer(
         "contributors",
         "Contributors",
-        "https://gitter.im/akka/dev",
+        "akka.official@gmail.com",
```

Comment:
do we have to have an email here?

Reply:
I just looked at this one: https://github.com/akka/akka/blob/main/project/Publish.scala#L44